readthedocs/doc_builder/director.py

```diff
@@ -418,10 +418,6 @@ def check_old_output_directory(self):
     def run_build_commands(self):
         """Runs each build command in the build environment."""
 
-        self.attach_notification(
-            message_id=BuildUserError.BUILD_COMMANDS_IN_BETA,
```

Comment:
Shouldn't this also be removed? I guess existing notifications use this message, but we could also just delete those.

Reply:
Yeah, existing notification use this message. We can decide what to do with them --but I think it's fine to keep them for now.